app/styles/app-dark.css

```diff
@@ -85,6 +85,19 @@ input,
     background: color-mod(var(--lightgrey));
 }
 
+input:focus,
+.gh-input:focus,
+.gh-select:focus,
+.gh-select select:focus {
+    border-color: color-mod(var(--darkgrey) l(-27%) blackness(15%));
```

Comment:
I'd suggest to use a little more subtle change (like we do on the regular/non-nightshift theme):
`border-color: color-mod(var(--lightgrey) l(+10%));`